lerobot/common/policies/act/modeling_act.py

```diff
@@ -38,7 +38,12 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module, PyTorchModelHubMixin):
+class ACTPolicy(nn.Module, PyTorchModelHubMixin, 
+                library_name="lerobot",
+                repo_url="https://github.com/huggingface/lerobot",
+                docs_url="https://github.com/huggingface/lerobot/",
+                tags=["robotics"],
```

Comment:
(not sure) Is `pipeline_tag: reinforcement-learning` appropriate for all policies? It yes, definitely good to add it to be listed on https://huggingface.co/models?pipeline_tag=reinforcement-learning 

```suggestion
                tags=["robotics"],
                pipeline_tag="reinforcement-learning",
```